conf.d/wmi_check.yaml.example

```diff
@@ -17,6 +17,10 @@ instances:
   #
   # `namespace` is the optionnal WMI namespace to connect to (default to `cimv2`).
   #
+  # `provider` is the optional WMI provider (default to `32` on Datadog Agent 32-bit or `64`).
+  # Usefull to request WMI data from the non-default provider. Available options are: `32` or `64`.
+  # For more information: https://msdn.microsoft.com/en-us/library/aa393067(v=vs.85).aspx
```

Comment:
I am wondering if we should not just make the provider default to `64` bit directly. By default, it looks like the WMI query would fallback to the 32-bit provider when no data is available.

If so, the `provider` option would not provide any more value and we could keep it undocumented for debugging purpose. 

What do you think ?